lightning/src/ln/wire.rs

```diff
@@ -56,13 +57,21 @@ pub enum Message {
 	GossipTimestampFilter(msgs::GossipTimestampFilter),
 	/// A message that could not be decoded because its type is unknown.
 	Unknown(MessageType),
+	Custom(T),
```

Comment:
What's the difference between `Unknown` and `Custom`?

Reply:
What's the difference? I don't see any RL code that will handle this differently, and I'm not entirely sure under what conditions we'd create such a thing?

Reply:
With this approach, Custom(T) is something that will be handled by the `UnknownMessageHandler` while `Unknown` will not be handled, and is necessary just to keep the code that check whether or not the message type is odd and should be ignored or even and it should error.

Reply:
Not relevant anymore given the new approach.